recipes/nimfa/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "nimfa" %}
+{% set version = "1.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 39cff2b86856d03ca8a3d9c38598034ecf1a768c325fd3a728bb9eadb8c6b919
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7|>=3.4
```

I guess you mean:
```suggestion
    - python =2.7|>=3.4
```
Without it's equivalent to `>=2.7` (`|` means "or").

---

Yes of course, sorry!